conjurapi/resource_test.go

```diff
@@ -26,7 +26,7 @@ func conjurSetup() (*Client, error) {
 
 - !permit
   role: !user alice
-  privilege: [ execute ]
+  privilege: [ execute, read ]
```

Comment:
Do we need to include `read` here, or can we run all our checks only on the `execute` privilege?